### DISCUSSION TITLE
Is it possible to contact bootstrap team please

### DISCUSSION THREAD

**Discussion by bhaveshjadav713** [twbs/bootstrap#39078] [Q&A]
Is it possible to contact bootstrap team please

Really i need help. Please reply me. Please sir Please Please Please

**Comment by julien-deramond** [comment]
You already are. [GitHub Discussions](https://github.com/orgs/twbs/discussions) are the best place to do it.

**Reply by bhaveshjadav713** [reply]
First of all thank you so much for your prompt reply.

I have a one question regarding the bootstrap theme create, can i ask you?

**Reply by julien-deramond** [reply]
Please create a [new GitHub Discussion](https://github.com/orgs/twbs/discussions/new/choose) with the right category, title, and description. Hopefully, someone in the community (core team or not) will be able to help you.

**Reply by bhaveshjadav713** [reply]
Ok, thank you so much for your faster reply

May i know which category is good for it?

I want to sell bootstrap theme on marketplace but i don't know how much time need to approve my account.

Waiting for your positive reply.

And Specially Thank you for your reply again and again Thank you..

**Reply by julien-deramond** [reply]
Oh OK, no need for a GitHub Discussion in this case, if the question is related only to Bootstrap Themes, this is not handled on GitHub since it's not maintained/provided by Bootstrap core team. I think you can contact them via their Zendesk Help Center: https://themes.zendesk.com/hc/en-us (link found in https://themes.getbootstrap.com/).

**Reply by bhaveshjadav713** [reply]
Sir already create ticket 10 days ago but no any reply from his side, check attachment.

**Edit: removed attachment to avoid personal information leak**

**Reply by bhaveshjadav713** [reply]
Hello sir, 

You there?

**Reply by julien-deramond** [reply]
Unfortunately, it's not handled by the Bootstrap Core Team here, so I can't help. To my knowledge, the only way to contact them is via this Zendesk platform.

**Reply by bhaveshjadav713** [reply]
Is there any other way to contact them if they are not reply via Zendesk platform. As i send screenshot 10days pass but no any reply. Do you know any expected reply time?

**Reply by bhaveshjadav713** [reply]
Please sir

Reply me.

**Reply by bhaveshjadav713** [reply]
Hii
